src/borg/archiver.py

```diff
@@ -1716,6 +1716,11 @@ def list_config(config):
                     if value is None:
                         raise Error('The repository config is missing the %s key which has no default value' % key)
                 print('%s = %s' % (key, value))
+            for key in ['last_segment_checked']:
```

Comment:
can you add a comma and a blank after the list entry? see 1662.

Reply:
hmm, the default value of last_segment_checked is (IIRC) 0 (or 1?).

if it is not there, it starts at the beginning, with the first segment.

Reply:
so the question is, whether this value could be treated in same way as the other values and not need special-casing with own code at all.

Reply:
I'd prefer to treat it specially. It should not be printed if it is not there. If we'd return last_segment_checked = 0 that means that a partial check is running, which is wrong.

Reply:
ok